weatherApp.js

```diff
@@ -122,7 +124,9 @@ windButton.addEventListener("click", renderWindContent);
 
 //zapukać do api i połączyć się z nim za pomocą XHR i promisów. Ew async await
 //zrobić listę danych: Miasto(nagłówek), niżej prognoza pogody na dziś, jutro, pojutrze w formacie `data-"rainy"`
-
+const upperFirstLetter = (name) => {
```

Comment:
From the way this param is named, I expect a single element. However, below you call name[0] suggesting it's an array.
This function should be called with a single string element, judging from the name (you call upper on string).